script/core/ActivateMarket.s.sol

```diff
@@ -72,4 +76,20 @@ contract ActivateMarket is Deployer {
 
     new UpdateMarket().innerRun({tkn0: tkn1, tkn1: tkn2, reader: reader});
   }
+
+  /**
+   * innerRun with no gasprice override (Mangrove's gasprice is used to compute density)
+   */
+  function innerRun(
+    Mangrove mgv,
+    MgvReader reader,
+    IERC20 tkn1,
+    IERC20 tkn2,
+    uint tkn1_in_gwei,
+    uint tkn2_in_gwei,
+    uint fee
+  ) public {
```

Comment:
Since this is the primary function signature for calling this script from other scripts, I think it should be above the special case with overrides + the doc should be complete.

Same for the corresponding function in `ActivateSemibook`.